curvefs/proto/copyset.proto

```diff
@@ -88,9 +88,17 @@ message CopysetsStatusResponse {
     repeated CopysetStatusResponse status = 1;
 }
 
+message GetAllCopysetsLoadStatusRequest {}
+
+message GetAllCopysetsLoadStatusResponse {
+    required bool loadFinish = 1;
+}
+
 service CopysetService {
     rpc CreateCopysetNode(CreateCopysetRequest) returns (CreateCopysetResponse);
     // TODO(chengyi): rm GetCopysetStatus
     rpc GetCopysetStatus(CopysetStatusRequest) returns (CopysetStatusResponse);
     rpc GetCopysetsStatus(CopysetsStatusRequest) returns (CopysetsStatusResponse);
+
+    rpc GetAllCopysetsLoadStatus(GetAllCopysetsLoadStatusRequest) returns (GetAllCopysetsLoadStatusResponse);
```

Comment:
Which component is this information exposed to？

Reply:
this has been removed.